### DISCUSSION TITLE
An intermediary gratitude

### DISCUSSION THREAD

**Discussion by psfinaki** [stryker-mutator/stryker-net#1812] [Show and tell]
Hey there, here I just wanted to separately thank Stryker team for all the work in the last months :) 

We in Microsoft are currently running Stryker against a codebase of around 150 projects and in about 90-95% cases Stryker runs successfully and consistently. We've moved everything to .NET 6 and C# 10 without much hassle as well. Mutation testing now affects basically all the developers working in this codebase and they have noticeably become much happier with Stryker, now respecting it as a quality gate and not as a flaky obstacle. By all this I mean that Stryker is not in "beta" anymore not just in regards to the version moniker but in the way it actually does its job.

That said, there are still some <s>bugs</s> challenges so expect more issues and asks in the near future :D Keep it up!

**Comment by rouke-broersma** [comment]
That is seriously awesome to hear! :)
